src/net.h

```diff
@@ -489,6 +490,7 @@ class CNode
     int grapheneBlockWaitingForTxns; // if -1 then not currently waiting
     CCriticalSection cs_grapheneadditionaltxs; // lock grapheneAdditionalTxs
     std::vector<CTransactionRef> grapheneAdditionalTxs; // entire transactions included in graphene block
+    uint32_t nGrapheneBloomfilterSize; // The maximum graphene bloom filter size (in bytes) that our peer will accept.
     uint64_t gr_shorttxidk0;
```

Comment:
You need to initialize the value of nGrapheneBloomFilterSize...this is done in the constructor for CNode() in net.cpp...

Reply:
Actually, I never ended up using that variable. I'll go ahead and remove it.